spec/actors/hyrax/actors/file_actor_spec.rb

```diff
@@ -131,112 +130,56 @@ class FileSetWithExtras < FileSet
         expect(Hyrax::VersioningService).to receive(:create).with(Hyrax::FileMetadata, user)
         expect(CharacterizeJob).to receive(:perform_later)
-        saved_metadata = actor.ingest_file(io)
-        reloaded = query_service.find_by(id: file_set.id)
-        expect(reloaded.member_by(use: relation).id).to eq saved_metadata.id
-      end
 
-      context 'when relation is a string' do
-        let(:relation) { 'thumbnail_file' }
+        metadata = actor.ingest_file(io)
 
-        it 'converts relation to URI before using the relation from the actor' do
-          pending 'implementation of Wings::Valkyrie::Persister #save_file_metadata'
-          expect(Hyrax::VersioningService).to receive(:create).with(Hyrax::FileMetadata, user)
-          expect(CharacterizeJob).to receive(:perform_later)
-          saved_metadata = actor.ingest_file(io)
-          reloaded = query_service.find_by(id: file_set.id)
-          expect(reloaded.member_by(use: relation).id).to eq saved_metadata.id
-        end
+        expect(Hyrax.custom_queries.find_many_file_metadata_by_use(resource: file_set, use: relation).map(&:id))
+          .to contain_exactly(metadata.id)
       end
     end
 
-    xit 'uses the provided mime_type' do
-      pending 'implementation of Wings::Valkyrie::Persister #save_file_metadata'
+    it 'uses the provided mime_type' do
       allow(fixture).to receive(:content_type).and_return('image/gif')
       expect(Hyrax::VersioningService).to receive(:create).with(Hyrax::FileMetadata, user)
       expect(CharacterizeJob).to receive(:perform_later)
-      saved_metadata = actor.ingest_file(io)
-      expect(saved_metadata.mime_type).to eq ['image/gif']
-    end
-
-    context 'with two existing versions from different users' do
-      let(:fixture2) { fixture_file_upload('/small_file.txt', 'text/plain') }
-      let(:huf2) { Hyrax::UploadedFile.new(user: user2, file: fixture2) }
-      let(:io2) { JobIoWrapper.new(file_set_id: file_set.id, user: user2, uploaded_file: huf2, path: huf2.uploader.path) }
-      let(:user2) { create(:user) }
-      let(:actor2) { described_class.new(file_set, relation, user2) }
-      let(:adapter) { Hyrax.metadata_adapter }
-      let(:query_service) { Wings::Valkyrie::QueryService.new(adapter: adapter) }
-      let(:versions) do
-        reloaded = query_service.find_by(id: file_set.id)
-        reloaded.original_file.versions
-      end
-
-      before do
-        # TODO: WINGS - When #ingest_file works, these should be uncommented.
-        # expect(Hyrax::VersioningService).to receive(:create).with(Hyrax::FileMetadata, user)
-        # expect(Hyrax::VersioningService).to receive(:create).with(Hyrax::FileMetadata, user2)
-        # expect(CharacterizeJob).to receive(:perform_later)
-        allow(CharacterizeJob).to receive(:perform_later)
-        actor.ingest_file(io)
-        actor2.ingest_file(io2)
-      end
 
-      it 'has two versions' do
```

Comment:
i'm having trouble maintaining this pending test (this is a ton of code!) so proposing removing it until we get versioning support on the valkyrie storage adapter.